src/components/BrowserCell/BrowserCell.react.js

```diff
@@ -214,12 +214,12 @@ export default class BrowserCell extends Component {
   //#endregion
 
   render() {
-    let { type, value, hidden, width, current, onSelect, onEditChange, setCopyableValue, setRelation, onPointerClick, row, col, field, onEditSelectedRow, readonly } = this.props;
+    let { type, value, hidden, width, current, onSelect, onEditChange, setCopyableValue, setRelation, onPointerClick, row, col, field, onEditSelectedRow, readonly, isRequired, markRequiredField } = this.props;
     let content = value;
     this.copyableValue = content;
     let classes = [styles.cell, unselectable];
     if (hidden) {
-      content = '(hidden)';
+      content = value !== undefined || row > -1 ? '(hidden)' : isRequired ? '(required)' : '(undefined)';
```

Comment:
You test for `row > -1` and below you test for `row < 0`. To make this more descriptive, can you introduce a variable like `let isNewRow = row < 0;` or whatever a descriptive name would be, and use that var in both conditions? It makes the code more readable.

Reply:
https://github.com/parse-community/parse-dashboard/pull/1720/commits/2d2dd2c91c9c502b28bd936de530f2b1c37f013e :)